src/sentry/eventstore/models.py

```diff
@@ -499,47 +496,6 @@ def get_raw_data(self, for_stream: bool = False) -> Mapping[str, Any]:
     def size(self) -> int:
         return len(orjson.dumps(dict(self.data)).decode())
 
-    def get_email_subject(self) -> str:
-        template = self.project.get_option("mail:subject_template")
-        if template:
-            template = EventSubjectTemplate(template)
-        elif self.group.issue_category == GroupCategory.PERFORMANCE:
```

Comment:
self.group is not available in `BaseEvent`. Moving this function to `EventGroup`.